components/PoisCard/PoiCard.vue

```diff
@@ -64,7 +64,14 @@
 
         {{ category }}
       </div>
+      <div :title="`${stars} ${$tc('fields.stars.label')}`" class="stars-data text-amber-500" v-if="stars">
 
+        <span :key="index" class="star-item" v-for="index in new Array(stars*1)">
+          <font-awesome-icon
+            :icon="'star'"
```

Comment:
But that does not use `Fields/Stars.vue` ?

Reply:
we can do this, i saw other fields of the poi card that were not using other components and thought it was what we do in this place.
will fix that

Reply:
Better to have an indeps component to test it.